src/tclint/commands/builtin.py

```diff
@@ -463,28 +463,56 @@ def _foreach(args, parser):
 
 
 def _if(args, parser):
-    # ref: https://www.tcl.tk/man/tcl/TclCmd/if.html
```

Comment:
nit: would prefer to keep this for consistency, although for accuracy it may be useful to take the chance to update this to https://www.tcl-lang.org/man/tcl8.6/TclCmd/if.htm instead (the link now goes to Tcl 9.0 docs, while these commands are meant to match 8.6).